core/templates/dev/head/components/summary_tile/ExplorationSummaryTileDirective.js

```diff
@@ -130,7 +131,16 @@ oppia.directive('explorationSummaryTile', [
             } else {
               var result = '/explore/' + $scope.getExplorationId();
               if ($scope.getCollectionId()) {
-                result += ('?collection_id=' + $scope.getCollectionId());
+                result = UrlService.addField(
+                  result, 'collection_id', $scope.getCollectionId());
+              }
+              if ($scope.getParentExplorationIds()) {
+                var parentExplorationIds = $scope.getParentExplorationIds();
+                for (var i = 0; i < parentExplorationIds.length - 1; i++ ) {
+                  result = UrlService.addField(result, 'parent',
```

Comment:
Contents of parens span multiple lines; break after first '('.

Reply:
Done, sorry about that.